oak/server/grpc_stream.h

```diff
@@ -14,41 +14,55 @@
  * limitations under the License.
  */
 
-#ifndef OAK_GRPC_SERVER_GRPC_STREAM_H_
-#define OAK_GRPC_SERVER_GRPC_STREAM_H_
+#ifndef OAK_GRPC_STREAM_H_
```

Comment:
Should this be `OAK_SERVER_GRPC_STREAM_H_` (after the full file name)?

Reply:
Done.